pkgs/development/python-modules/pyobjc-core/default.nix

```diff
@@ -0,0 +1,133 @@
+# inpired by https://github.com/input-output-hk/lace/blob/0fd166666cc454171811b307050d0815452bea82/nix/lace-blockchain-services/internal/any-darwin.nix#L339
+{
+  lib,
+  pkgs,
+  stdenv,
+  buildPythonPackage,
+  fetchPypi,
+  xcodebuild,
+  cctools,
+  darwin,
+}:
+let
+  appleSDK = darwin.apple_sdk_11_0;
+  inherit (appleSDK) MacOSX-SDK objc4;
+  darwinFrameworks = appleSDK.frameworks;
+  inherit (darwinFrameworks) Foundation GameplayKit MetalPerformanceShaders;
+  # if attempting to use libffi from pkgs this patching is required
+
+  # grep -RF '<ffi/ffi.h>' | cut -d: -f1 | while IFS= read -r file ; do
+  #   sed -r "s+<ffi/ffi.h>+<ffi.h>+g" -i "$file"
+  # done
+
+  # and the following errors appear
+
+  # Modules/objc/libffi_extra.m:94:17: error:
+  # call to undeclared function 'ffi_find_closure_for_code_np';
+  # ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
+  # Modules/objc/libffi_extra.m:94:15: error:
+  # incompatible integer to pointer conversion assigning to 'ffi_closure *' from 'int' [-Wint-conversion]
+  # Using libffi from MacOS SDK solves this issue
+
+  # TODO: This is the most straightforward way to fis the build yet it is not entirely obvious if
+  # apple_libffi deserves its own package or can be safely replaced with libffi with some patches or workarounds
+  apple_libffi = stdenv.mkDerivation {
+    name = "apple-libffi";
```

Comment:
Use pname and inherit the version please

Reply:
did that